js/app.js

```diff
@@ -1 +1,121 @@
-console.log('Hello frontend');
+// Logic Reference: https://github.com/angle943/tic-tac-toe
+// have bugs when meet condition 2=5=8, 3=6=9, 4=5=6, 7=8=9
+// add if(one) etc. to check if it is empty ''
+
+// Variables
+
+msg = document.querySelector('#message h2')
+xScores = document.getElementById('xScores')
+oScores = document.querySelector('#oScores')
+tieScores = document.querySelector('#tieScores')
+let xWins = 0
+let oWins = 0
+let tie = 0
+let moves = 0
+let next = ''
+let gameIsGoing = true
+console.log(xScores, oScores);
+
+// Functions
+
+// display winner
+function display(player) {
+    if (player === 'X'){
+        msg.textContent = 'X Wins!!!'
+        gameIsGoing = !gameIsGoing
+        xWins ++
+        xScores.textContent = xWins
+    }
+    else if (player === 'O') {
+        msg.textContent = 'O Wins!!!'
+        gameIsGoing = !gameIsGoing
+        oWins ++
+        oScores.textContent = oWins
+    }
+    // some boxs have '', which also meet the if statement, took time to figure out...
+    // else {
+    //     msg.textContent = next + ' is next'
+    // }
+}
+
+// check winning cases
+function gameStatus(){
+    //.textContent work just the same like .classList[1], since I name the additional class 'X' or 'O'
+    one = document.querySelector('#one').classList[1]
+    two = document.querySelector('#two').classList[1]
+    three = document.querySelector('#three').classList[1]
+    four = document.querySelector('#four').classList[1]
+    five = document.querySelector('#five').classList[1]
+    six = document.querySelector('#six').classList[1]
+    seven = document.querySelector('#seven').classList[1]
+    eight = document.querySelector('#eight').classList[1]
+    nine = document.querySelector('#nine').classList[1]
+
+    if (one && one === two && one === three) {display(one)}
```

Comment:
Just a quick check that you understand why your check here includes if (one...) rather than just the checks to see if one === two && one === three.  I saw the project you referenced at the top of the document also does this and want to make sure it makes sense to you.  Do you know why you are adding the first number in each of these if/else if statements?